README.md

```diff
@@ -5,7 +5,7 @@ https://bridge-monitoring.poa.net/
 This tools allows you to spin up node.js server to monitor for 2 contracts on
 Home and Foreign Eth networks to check for balance difference.
 On Home network it checks for `HOME_BRIDGE_ADDRESS` balance
-On Foreign network it checks for `POA20_ADDRESS` total supply.
+On Foreign network it checks for `ERC20_ADDRESS` total supply.
```

Comment:
We should update `.env.example` too

Reply:
You are right, I missed that.

Reply:
Done.

Reply:
So it could be corrected now, right?

Reply:
Done. Please check if wording is Ok @akolotov